decidim-admin/app/packs/src/decidim/admin/managed_moderated_users.js

```diff
@@ -0,0 +1,147 @@
+/* eslint-disable no-invalid-this */
```

Comment:
Being a new JS file, please Update the syntax to vanilla JS.

Reply:
Done